app/controllers/lectures_controller.rb

```diff
@@ -61,10 +50,7 @@ def update
 
   # DELETE /lectures/1
   def destroy
-    if current_user.is_student?
-      redirect_to lectures_url, notice: "You are a student! You can not delete a lecture :("
-    else
-      @lecture.destroy
+    if @lecture.destroy
```

Comment:
Maybe only the lecturer that created it should be able to start, end, delete a lecture? Not sure though, not specified in the issue.

Reply:
I think it would make sense to add it, even though it wasn't explicetly mentioned.